### ISSUE TITLE
The training detail of PillarNext with FPN as its neck module

### ISSUE THREAD

**Issue by ouyangziyao** [qcraftai/pillarnext#28]
Hi Sir,

Thank you for your excellent work. It has been incredibly helpful for me. However, after reading your great paper, I have a question regarding the training details of PillarNext with FPN as its neck module. In a typical FPN, the feature maps in the top-to-bottom path are all utilized to predict their respective objects (as shown in this picture). For PillarNext with FPN, does the training utilize all the feature maps or just a single feature map?

![image](https://github.com/user-attachments/assets/009e3786-e07f-4308-aabc-ee1ac1c60601)

**Comment by Konstantin5389** [comment]
No, we just use FPN as a feature fusion method. We use center head for fair comparison.
